.github/workflows/ci.yml

```diff
@@ -44,6 +51,15 @@ jobs:
         go-version: 1.20.x
         check-latest: true
         cache: true
+    - name: Set PLUGINS env var from changed files
+      env:
+        ALL_MODIFIED_FILES: ${{ steps.changed-files.outputs.all_modified_files }}
+        ANY_MODIFIED: ${{ steps.changed-files.outputs.any_modified }}
+      run: |
+        val=`go run ./internal/cmd/changed-plugins .`
+        if [[ -n "${val}" && -z "${PLUGINS}" ]]; then
+          echo "PLUGINS=${val}" >> $GITHUB_ENV
+        fi
```

Comment:
This runs the new utility that uses the github actions variables to determine which plugins have changed. This only runs if someone didn't manually set PLUGINS to a valid value above.